easydita-knowledge-portal/inc/customizer.php

```diff
@@ -125,6 +125,32 @@ public function render_content() {
 		)
 	);
 
+  /** Page Content **/
+  $wp_customize->add_section(
+    'page_content',
+    array(
+      'title' => __('Page Content', 'easydita_knowledge_portal'),
+      'description' => __('Customize layout of content on page.', 'easydita_knowledge_portal'),
+      'priority' => 40
+    )
+  );
+  $wp_customize->add_setting(
+    'move_short_desc',
+    array(
+      'default' => "default",
```

Comment:
The default should be "0" I think because that's what you use down when getting the value. (If it's not defined, it's 0.) But I'm not exactly sure. As long as it's showing up as not checked for the first time, that's fine.